### ISSUE TITLE
Support for webpack 5

### ISSUE THREAD

**Issue by connor4312** [eamodio/vscode-tsl-problem-matcher#4]
Webpack 5 change its output format, it doesn't look like the problem matcher works for it.

Example output:

```
> Executing task in folder extension: npm run watch <


> css-theme-completions@0.0.1 watch C:\Users\Connor\Documents\Github\vscode-theme-completions\extension
> webpack --watch --mode development

[webpack-cli] Compilation starting...
[webpack-cli] Compilation starting...
(node:18984) [DEP_WEBPACK_WATCH_WITHOUT_CALLBACK] DeprecationWarning: A 'callback' argument need to be provided to the 'webpack(options, callback)' function when the 'watch' option is set. There is no way to handle the 'watch' option without a callback.
[webpack-cli] Compilation finished
asset extension.node.js 57.7 KiB [compared for emit] (name: main)
runtime modules 931 bytes 4 modules
built modules 44.1 KiB [built]
  cacheable modules 43.9 KiB
    ./src/extension.ts 3.75 KiB [built] [code generated]
    ./node_modules/node-fetch/lib/index.mjs 40.1 KiB [built] [code generated]
  external "vscode" 42 bytes [built] [code generated]
  external "stream" 42 bytes [built] [code generated]
  external "http" 42 bytes [built] [code generated]
  external "url" 42 bytes [built] [code generated]
  external "https" 42 bytes [built] [code generated]
  external "zlib" 42 bytes [built] [code generated]
webpack 5.3.2 compiled successfully in 2857 ms

asset extension.web.js 9.9 KiB [compared for emit] (name: main)
runtime modules 931 bytes 4 modules
built modules 3.83 KiB [built]
  ./src/extension.ts 3.75 KiB [built] [code generated]
  external "fetch" 42 bytes [built] [code generated]
  external "vscode" 42 bytes [built] [code generated]
webpack 5.3.2 compiled successfully in 2662 ms
[webpack-cli] watching files for updates...

```

**Comment by eamodio** [comment]
@connor4312 Which matcher are you using? I am currently using this with webpack 5 and it works for me (though we could certainly have different setups)

**Comment by connor4312** [comment]
This is in https://github.com/connor4312/vscode-css-theme-completions/tree/main/extension with ts-webpack. I can look into it a bit this weekend if you aren't hitting it.

**Comment by eamodio** [comment]
Hrm, I'll check it out for ts-loader alone, I've always been using `fork-ts-checker` lately.

**Comment by SteveBenz** [comment]
I can't say when this change happened, but I have webpack 5.11 and the message that it puts out when it completes compilation unsuccessfully no longer matches the pattern.  It's also true that 5.11 doesn't have a --info-verbosity command line option, so that chunk should be updated in the readme.

**Comment by felipecrs** [comment]
For those subscribed to this issue, you may be interested in https://github.com/eamodio/vscode-tsl-problem-matcher/pull/9#issuecomment-944955718.

**Comment by eamodio** [comment]
This should be fixed in v0.5.0
